### ISSUE TITLE
关于网站logo的问题

### ISSUE THREAD

**Issue by MayuOshima** [miccall/hexo-theme-Mic_Theme#23]
为什么我把配置文件里的favicon的图片链接改成我自己的图片后，显示的logo还是你那个啊？
我把img和images里你的logo删了并且把我的logo的图片改成你原来logo的图片的名字，可是还是不行

**Comment by sli1989** [comment]
LOGO is set in `themes\miccall\source\img`.
MAKE SURE the type of logo file is the same in the `_config.yml`.

**Comment by MayuOshima** [comment]
yes I just did what you say, but the logo still appears to be yours or sometimes there is no logo

----- 原始邮件 -----
发件人: "Alex LEE" <notifications@github.com>
收件人: "miccall/hexo-theme-Mic_Theme" <hexo-theme-Mic_Theme@noreply.github.com>
抄送: "Ajin" <magi-yunan@sjtu.edu.cn>, "Author" <author@noreply.github.com>
发送时间: 星期六, 2018年 1 月 13日 下午 7:20:11
主题: Re: [miccall/hexo-theme-Mic_Theme] 关于网站logo的问题 (#23)

LOGO is set in `themes\miccall\source\img`.
MAKE SURE the type of logo file is the same in the `_config.yml`.

**Comment by sli1989** [comment]
did you hexo clean before hexo g ? have a try...
where can see your public files?

**Comment by MayuOshima** [comment]
yes I have done hexo clean.
mayuoshima.github.io is my gitpage and now I am using your theme.
(emmmm...sorry to disturb me so much, if there remains the problem, actually that's fine...)

----- 原始邮件 -----
发件人: "Alex LEE" <notifications@github.com>
收件人: "miccall/hexo-theme-Mic_Theme" <hexo-theme-Mic_Theme@noreply.github.com>
抄送: "Ajin" <magi-yunan@sjtu.edu.cn>, "Author" <author@noreply.github.com>
发送时间: 星期六, 2018年 1 月 13日 下午 7:55:09
主题: Re: [miccall/hexo-theme-Mic_Theme] 关于网站logo的问题 (#23)

did you hexo clean before hexo g ? have a try...
where can see your public files?

**Comment by sli1989** [comment]
see?  you have NOT put the logo file in `themes\miccall\source\img` from [the public fies](https://github.com/MayuOshima/mayuoshima.github.io/tree/master/img).

**Comment by MayuOshima** [comment]
???? I can see my logo image in the url you say...

----- 原始邮件 -----
发件人: "Alex LEE" <notifications@github.com>
收件人: "miccall/hexo-theme-Mic_Theme" <hexo-theme-Mic_Theme@noreply.github.com>
抄送: "Ajin" <magi-yunan@sjtu.edu.cn>, "Author" <author@noreply.github.com>
发送时间: 星期六, 2018年 1 月 13日 下午 9:15:18
主题: Re: [miccall/hexo-theme-Mic_Theme] 关于网站logo的问题 (#23)

see?  you have NOT put the logo file in `themes\miccall\source\img` from [the public fies](https://github.com/MayuOshima/mayuoshima.github.io/tree/master/img).

**Comment by sli1989** [comment]
omg...
there is a apace between the name and `.`, delete it?
and you dot need to replace the file, just add a new file and replace the `favicon: "/img/xx.xx"`.

**Comment by MayuOshima** [comment]
oh! that's the case!
thank you 

----- 原始邮件 -----
发件人: "Alex LEE" <notifications@github.com>
收件人: "miccall/hexo-theme-Mic_Theme" <hexo-theme-Mic_Theme@noreply.github.com>
抄送: "Ajin" <magi-yunan@sjtu.edu.cn>, "Author" <author@noreply.github.com>
发送时间: 星期六, 2018年 1 月 13日 下午 9:35:15
主题: Re: [miccall/hexo-theme-Mic_Theme] 关于网站logo的问题 (#23)

omg...
there is a apace between the name and `.`, delete it?
and you dot need to replace the file, just add a new file and replace the `favicon: "/img/xx.xx"`.
